R/utils.R

```diff
@@ -550,3 +550,10 @@ chunk_by_cols <- function(data, id_per_chunk, cols) {
 }
 
 force_digits <- function(x,digits) formatC(x,digits=digits,format = 'f')
+
+require_patchwork <- function() {
```

Comment:
Fine as is, but `pm_grid` could now use `require_patchwork`.

Reply:
Done.